packages/components/select/src/useSelect.ts

```diff
@@ -407,7 +407,11 @@ export const useSelect = (props, states: States, ctx) => {
         originClientHeight ||
         (input.clientHeight > 0 ? input.clientHeight + 2 : 0)
       const _tags = tags.value
-      const gotSize = getComponentSize(selectSize.value || form?.size)
+      const cssVarOfSelectSize =
+        getComputedStyle(input).getPropertyValue('--el-input-height')
+      const gotSize = cssVarOfSelectSize
+        ? cssVarOfSelectSize
+        : getComponentSize(selectSize.value || form?.size)
```

Comment:
```suggestion
       const gotSize = cssVarOfSelectSize || getComponentSize(selectSize.value || form?.size)
```

Reply:
@kamesan012  You cannot directly commit this line of modification, and the above two lines of code should also be deleted.

Reply:
Thanks.